selfapi.js

```diff
@@ -367,6 +372,21 @@ API.prototype = {
     });
   },
 
+  // Build index routes.
+  toAPIIndex: function (basePath) {
```

Comment:
`toHTML` uses paths in examples like `POST /api/hosts`, whereas [exportAllTests](https://github.com/JanitorTechnology/selfapi/blob/6cad7814e2bbfd4f892e25dc40f3237ca5b2a6da/selfapi.js#L124) uses URLs like `http://localhost:8080/api/hosts` to actually test the server.

It all depends how you want the exported API Index to look like, would it be better with paths or full URLs?